custom/logistics/api.py

```diff
@@ -153,6 +154,9 @@ def set_default_backend(self):
         domain_object.default_sms_backend_id = MobileBackend.load_by_name(None, 'MOBILE_BACKEND_TEST').get_id
         domain_object.save()
 
+    def create_or_edit_roles(self):
+        raise NotImplemented("Not implemented yet")
```

Comment:
small but seems fine for this to do nothing in the base class if only one object needs it